### PR TITLE
Dashboard plugin handover

### PR DESCRIPTION
**Pull request by j7126**
Updates for the dashboard plugin transferring to new maintainers.

### CODE REVIEW

**Comment by j7126** [comment]
Resolve abandoned plugin issue #628

**Comment by jneilliii** [comment]
Hey guys, thanks for adopting this plugin. Just wanted to ask a question in regard to the author. Historically the new maintainers have been appended to the author field so I wanted to check with you first to see if that was intentional or not?

**Comment by jneilliii** [comment]
One other thing. Would you guys mind migrating your images into the PR as well rather than linking those images to github? This is a new thing that we're trying to clean up to limit GDPR related concerns.

File: _plugins/dashboard.md [hunk 1]
```diff
@@ -20,45 +20,53 @@ tags:
 - progress
 
 screenshots:
-- url: https://github.com/StefanCohen/OctoPrint-Dashboard/raw/master/screenshot.png
+- url: https://github.com/j7126/OctoPrint-Dashboard/raw/master/screenshot.png
```
Comment (jneilliii) [inline _plugins/dashboard.md:23]: update to the following and copy relative screenshot.
```
/assets/img/plugins/dashboard/screenshot.png
```

File: _plugins/dashboard.md [hunk 2]
```diff
@@ -20,45 +20,53 @@ tags:
...
   alt: screenshot
-  caption: caption of a screenshot
-- url: https://github.com/StefanCohen/OctoPrint-Dashboard/raw/master/screenshot-theme.png
+  caption: Dashboard screenshot
+- url: https://github.com/j7126/OctoPrint-Dashboard/raw/master/screenshot-theme.png
```
Comment (jneilliii) [inline _plugins/dashboard.md:26]: update to the following and copy relative screenshot.
```
/assets/img/plugins/dashboard/screenshot-theme.png
```

File: _plugins/dashboard.md [hunk 3]
```diff
@@ -20,45 +20,53 @@ tags:
...
   alt: screenshot
-  caption: themed screenshot
-- url: https://github.com/StefanCohen/OctoPrint-Dashboard/raw/master/screenshot-theme2.png
+  caption: Themed dashboard screenshot
+- url: https://github.com/j7126/OctoPrint-Dashboard/raw/master/screenshot-theme2.png
```
Comment (jneilliii) [inline _plugins/dashboard.md:29]: update to the following and copy relative screenshot.
```
/assets/img/plugins/dashboard/screenshot-theme2.png
```

File: _plugins/dashboard.md [hunk 4]
```diff
@@ -20,45 +20,53 @@ tags:
...
   alt: screenshot
-  caption: themed screenshot
+  caption: Themed dashboard screenshot
 
-featuredimage: https://github.com/StefanCohen/OctoPrint-Dashboard/raw/master/screenshot.png
+featuredimage: https://github.com/j7126/OctoPrint-Dashboard/raw/master/screenshot.png
```
Comment (jneilliii) [inline _plugins/dashboard.md:33]: update to the following
```
/assets/img/plugins/dashboard/screenshot.png
```

File: _plugins/dashboard.md [hunk 5]
```diff
@@ -20,45 +20,53 @@ tags:
...
 
 compatibility:
   python: ">=2.7,<4"
 
-abandoned: https://github.com/OctoPrint/plugins.octoprint.org/issues/628
 ---
 This plugin adds a  dashboard tab in Octoprint that displays the most relevant info regarding the state of the printer and any on-going print job.
 
-![Screenshot](https://github.com/StefanCohen/OctoPrint-Dashboard/raw/master/screenshot.png)
+![Screenshot](https://github.com/j7126/OctoPrint-Dashboard/raw/master/screenshot.png)
```
Comment (jneilliii) [inline _plugins/dashboard.md:41]: update to the following
```
![Screenshot](/assets/img/plugins/dashboard/screenshot.png)
```

File: _plugins/dashboard.md [hunk 6]
```diff
@@ -20,45 +20,53 @@ tags:
...
 * Supports chamber temperature if configured in the printer profile
 * Configurable progress gauge type (Circle, Bar)
+* Fullscreen mode including job control buttons (Start, Cancel, Pause/Resume)
+* Full page mode by adding `?dashboard=full` parameter at the end of the octoprint url
 * Uses Estimates from [PrintTimeGenius](https://plugins.octoprint.org/plugins/PrintTimeGenius/) when installed
 * Uses GCode analysis provided by [DisplayLayerProgress](https://plugins.octoprint.org/plugins/DisplayLayerProgress/) to get more accurate layer and fan data
 * Theme friendly:
 
-![Screenshot](https://github.com/StefanCohen/OctoPrint-Dashboard/raw/master/screenshot-theme.png)
+![Screenshot](https://github.com/j7126/OctoPrint-Dashboard/raw/master/screenshot-theme.png)
```
Comment (jneilliii) [inline _plugins/dashboard.md:69]: update to the following
```
![Screenshot](/assets/img/plugins/dashboard/screenshot-theme.png)
```

**Review by jneilliii — CHANGES_REQUESTED** [review]
@j7126 relative to the GDPR related concerns I mentioned, please see the relative comments and associated changes. Basically just need to get those images copied to `assets/img/plugins/dashboard/` on your PR and update the references in the markdown file as noted please. Thanks for the help on this.

**Comment by j7126** [comment]
Will do. I am just preparing to redo all the screenshots because they are from an older version of the plugin.

**Comment by jneilliii** [comment]
Oh, my bad. I was trying to help you out a little and do these changes for you, and just added to this PR. I apologize if this breaks something in the git workflow/commit history.

**Comment by j7126** [comment]
Added new screenshots.
Should be ready to merge now

**Review by jneilliii — APPROVED** [review]

**Pull request merged**